readme.rst

```diff
@@ -169,7 +169,7 @@ Configure the spelling check
 If your documentation uses US English instead of UK English, change this in the
 ``.sphinx/spellingcheck.yaml`` file.
 
-To add exceptions for words the spelling check marks as wrong even though they are correct, edit the ``.custom_wordlist.txt`` file. 
+To add exceptions for words the spelling check marks as wrong even though they are correct, edit the ``.custom_wordlist.txt`` file.
```

Comment:
I think this reads better:
To add exceptions for words flagged by spelling check, edit the ``.custom_wordlist.txt`` file.